app/services/puppet_class_importer.rb

```diff
@@ -88,7 +88,7 @@ def updated_classes_for(environment)
     return [] unless db_environments.include?(environment) && actual_environments.include?(environment)
     HashWithIndifferentAccess[
       db_classes(environment).map do |db_class|
-        params = EnvironmentClass.all_parameters_for_class(db_class.id, find_or_create_env(environment).id).map(&:lookup_key)
+        params = EnvironmentClass.all_parameters_for_class(db_class.id, find_or_create_env(environment).id).map(&:puppetclass_lookup_key)
```

Comment:
I changed the map to pluck in the controllers bu I think the importer should be in a separate PR with maybe more performance fixes there?